<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>700</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>750</width>
    <height>650</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Visionox | VxD - EZ-PMOLED</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Size">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>128 x128</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Driver">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IC1234</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Type">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mono or Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LoadImage">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Filename">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>No file loaded!</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_OLEDImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QLabel" name="label_Status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VxD OLEDHAT - Loading...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_CHeaderOutput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Image History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveAsCCode">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>580</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as C Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Size (Pixels)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Driver IC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_History">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>M0039F</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_JoyStick">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
